### DISCUSSION TITLE
Scroll page per page with animation in Scrollviewer?

### DISCUSSION THREAD

**Discussion by dbriard** [AvaloniaUI/Avalonia#11683] [Q&A]
Hi, I have a scrollviewer containing an itemsrepeater with a list of items, and I would like to scroll page per page with an animated scroll (triggered bu left/right buttons)

Is there already something in the API to do that? or a sample? or a recommended way to implement a ScrollTo with animation?

I know there is carousel but it only work with 1 view at the time, and I have an itemsrepeater with several items per page.

Thanks for guidance.

**Comment by Egolds** [comment]
Check this: https://github.com/Egolds/Avalonia.Xaml.Interactions.Animated

**Reply by dbriard** [reply]
@Egolds Thanks a lot, I finally get it working, but I'll have a look at your project!
